<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendFile</class>
 <widget class="QWidget" name="SendFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Nightwood - Send File</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <layout class="QHBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="NavBar" name="navBar" native="true">
     <property name="minimumSize">
      <size>
       <width>220</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>220</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="contentLayout">
     <property name="spacing">
      <number>25</number>
     </property>
     <property name="leftMargin">
      <number>35</number>
     </property>
     <property name="topMargin">
      <number>35</number>
     </property>
     <property name="rightMargin">
      <number>35</number>
     </property>
     <property name="bottomMargin">
      <number>35</number>
     </property>
     <item>
      <widget class="QLabel" name="pageTitle">
       <property name="styleSheet">
        <string notr="true">font-size: 24px; font-weight: bold; color: #2d3436;</string>
       </property>
       <property name="text">
        <string>Send File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="formContainer" native="true">
       <property name="styleSheet">
        <string notr="true">QWidget#formContainer {
  background-color: white;
  border: 1px solid #dfe6e9;
  border-radius: 12px;
  padding: 25px;
}
QLineEdit {
  padding: 8px 12px;
  font-size: 14px;
  border-radius: 6px;
  background-color: white;
  border: 1px solid #dfe6e9;
  color: #2d3436;
  margin: 4px 0;
  min-height: 20px;
}
QLineEdit:focus {
  border: 2px solid #6c5ce7;
}
QLabel {
  color: #2d3436;
  font-weight: bold;
  font-size: 14px;
  margin: 8px 0 4px 0;
}
QPushButton#browseButton {
  color: #2d3436;
  background-color: white;
  border: 1px solid #dfe6e9;
  border-radius: 6px;
  min-height: 32px;
  max-height: 32px;
  font-size: 13px;
  padding: 4px 12px;
}
QPushButton#browseButton:hover {
  background-color: #f1f2f6;
  border-color: #6c5ce7;
  color: #6c5ce7;
}
QLabel#fileDetailsLabel {
  font-size: 13px;
  font-weight: normal;
  color: #636e72;
  padding: 12px;
  background-color: #f8f9fa;
  border: 1px solid #dfe6e9;
  border-radius: 6px;
  margin: 8px 0;
  line-height: 1.4;
}</string>
       </property>
       <layout class="QVBoxLayout" name="formLayout">
        <property name="spacing">
         <number>15</number>
        </property>
        <property name="leftMargin">
         <number>25</number>
        </property>
        <property name="topMargin">
         <number>25</number>
        </property>
        <property name="rightMargin">
         <number>25</number>
        </property>
        <property name="bottomMargin">
         <number>25</number>
        </property>
        <item>
         <widget class="QLabel" name="usernameLabel">
          <property name="text">
           <string>Send to username:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="usernameInput">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>42</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>Enter recipient's username</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="fileLabel">
          <property name="text">
           <string>Select File:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="fileLayout">
          <property name="spacing">
           <number>8</number>
          </property>
          <item>
           <widget class="QLineEdit" name="filePathInput">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>42</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="placeholderText">
             <string>Choose a file to send</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="browseButton">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>42</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>42</height>
             </size>
            </property>
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="fileDetailsLabel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>120</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QPushButton" name="sendButton">
          <property name="minimumSize">
           <size>
            <width>130</width>
            <height>44</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
  font-size: 14px;
  font-weight: bold;
  color: white;
  background-color: #6c5ce7;
  border-radius: 10px;
  padding: 0 10px;
}
QPushButton:hover {
  background-color: #5049c9;
}
QPushButton:pressed {
  background-color: #4040b0;
}</string>
          </property>
          <property name="text">
           <string>Send File</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NavBar</class>
   <extends>QWidget</extends>
   <header>src/ui/utils/navbar/navbar.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
